JH6_TestNG/src/test/java/homework/transport/BicycleTest.java

```diff
@@ -0,0 +1,44 @@
+package homework.transport;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class BicycleTest {
+
+    Bicycle bicycle;
+    Router router;
+
+    @BeforeMethod
+    public void setUp() throws Exception {
+        bicycle = new Bicycle(2.0);
+        router = new Router();
+        router.setStartCoordinates(0.0, 0.0);
```

Comment:
You should check the cases with all accepted start point (with respect to eq. classes)